content/02.main-text.md

```diff
@@ -1,24 +1,72 @@
+
+## The Deep Review
+
+We initiated the Deep Review by creating a GitHub repository (https://github.com/greenelab/deep-review) to coordinate and manage contributions.
+GitHub is a platform designed for collaborative software development that's adaptable for collaborative writing.
+From the start, we made the GitHub repository public, applying a Creative Commons Attribution [License](https://github.com/greenelab/deep-review/blob/master/LICENSE.md) to the manuscript.
+Next, we encouraged anyone interested to contribute by proposing changes or additions.
+Although we requested that some authors participate for their specific expertise, most discovered the manuscript organically through conferences or social media, deciding without solicitation to contribute.
+In total, the Deep Review attracted {{deep_review_authors}} authors from 20 different institutions (and counting) who were not determined in advance.
+
+To coordinate this effort, we developed a manuscript writing process using the Markdown language, [GitHub](https://github.com/greenelab/deep-review/), and our new [Manubot](https://github.com/greenelab/manubot) tool for automating manuscript generation.
+
+## Results
+
+Writing reviews in a public forum facilitates engage with the authors of original research to clarify their methods and results and present them accurately, as exemplified [here](https://github.com/greenelab/deep-review/issues/213).
+
 `TODO: need archival issue link`
 In addition, discussing manuscripts in the open provides one form of pre- and post-publication peer review `TODO: define this or provide a reference?`, incentivizing the reviews with potential manuscript authorship.
 However, inviting wide authorship brings many technical and social challenges such as how to fairly distribute credit, coordinate the scientific content, and collaboratively manage extensive reference lists.
 
-We present solutions to these challenges based on our recent experience leading a collaborative review "Opportunities And Obstacles For Deep Learning In Biology And Medicine" [@doi:10.1101/142760].
-Our review attracted {{deep_review_authors}} authors from 20 different institutions who were not determined in advance.
-We wrote entirely in the open without restrictions on who was welcome to contribute.
-Although we requested that some authors participate for their specific expertise, most discovered the manuscript organically through conferences or social media and independently decided to contribute.
-To coordinate this effort, we developed a manuscript writing process using the Markdown language, the [GitHub software development platform](https://github.com/greenelab/deep-review/), and our new [Manubot](https://github.com/greenelab/manubot) tool for automating manuscript generation.
+## Contribution workflow
```

Comment:
I like this coming earlier, but I wonder if our dive into the technical side is too deep too fast. I don't think it should hold up this PR but we may want to consider how accessible this is to a general audience as we move forward.

Reply:
Yeah, I moved the text around since before there was poor flow from between the different aspects of the paper (collaborative reviews, deep reviews, manubot). There's definitely still work to do in this regard after this PR is merged.